src/data/icons/svg/OUSD.svg

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 794 792" style="enable-background:new 0 0 794 792;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#1E313F;}
+	.st1{opacity:0.2;fill:#FFFFFF;}
+	.st2{fill:#FAFBFC;}
+	.st3{fill:url(#SVGID_1_);}
+	.st4{fill:url(#SVGID_2_);}
+</style>
```

Comment:
we can't afford to have `id=`, `class=` or `<style>` in these SVG because we will convert them back into react class. Thanks

Reply:
Thanks @gre have changed the icon to not include `id`, `class` or `<style>` tags